packages/html/src/dialog/templates/dialog-modal.tsx

```diff
@@ -0,0 +1,13 @@
+import Dialog from "../dialog.spec";
+
+export const DialogModal = ({ children, ...other }: any) => (
+    <>
+        <div className="k-dialog-wrapper">
+            <div className="k-overlay"></div>
```

Comment:
any specific reason why this is not part of the spec ?

Reply:
**Comment Update:**
We had a discussion and made the decision to implement it at a later time.